Wikipedia/UI-V5/WMFLocationManager.m

```diff
@@ -6,8 +6,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-static CLLocationDistance WMFMinimumDistanceBeforeUpdatingLocation = 1.0; //meters before we update location
-
 @interface WMFLocationManager ()<CLLocationManagerDelegate>
```

Comment:
I can see one thing that changed… we are no longer caching the last location… which was intentional… because the current location in the CLLocation manager can be nilled out like when starting and stopping location updates. So now there is now time when the location can be nil in the between refreshes… when before that wasn't true.